src/utils/FontManager.js

```diff
@@ -114,7 +114,13 @@ export async function fixupColorFonts() {
         // weights for the emoji to appear in bold messages, etc.
         document.fonts.add(new FontFace("Twemoji", path, { weight: 600 }));
         document.fonts.add(new FontFace("Twemoji", path, { weight: 700 }));
+    } else {
+        // fall back to SBIX, generated via https://github.com/matrix-org/twemoji-colr/tree/matthew/sbix
```

Comment:
it's very unfortunate that we're not able to use upstream for this. I have many fears about us forgetting to upgrade, or falling behind, or otherwise not maintaining our fork very well.

Reply:
yup, but i see no alternative sadly.